src/Controller/ProfileClientController.php

```diff
@@ -15,6 +15,7 @@ class ProfileClientController extends AbstractController
      */
     public function index(): Response
     {
+        $this->denyAccessUnlessGranted('ROLE_ADMIN');
```

Comment:
pareil, je ne pense pas que ce soit bon de mettre un ROLE_ADMIN comme ça.
De mémoire, on avait dit qu'un client pouvait voir sa propre page (et pas celle des autres) mais qu'un ADMIN pouvait lui voir toutes les pages ? 
donc on veut un /profile/client/{id}  avec cette id qui est optionnel, et SI il est là, alors on doit etre ADMIN, et le user affiché est celui de l'id, sinon l'user est récupérer avec $this->getUser()

Reply:
Oui c'est ça, c'est juste que nous on appelle notre admin "superadmin" et que les clients et coach ont le rôle "admin"